### ISSUE TITLE
[Bug] Page is not scrollable after router.push if modal was not closed

### ISSUE THREAD

**Issue by aleksanderzubkov** [quasarframework/quasar#1443]
### What did you get as the error?
If modal was not closed before router.push() then on next router page BODY remains class ".with-modal" and it's not possible to scroll page.

Bug was found after upgrade to 0.14.7. Version 0.14.4 did not has this bug. 

### What were you expecting?
Page should be scrollable.

As temporary fix I use router guard:

> router.afterEach((to, from) => {
>     document.body.classList.remove('with-modal');
> });

**Comment by rstoenescu** [comment]
Hi,

It's in docs. Modals affect window history. You need to close one before navigating away or opening another. Applies to all components that mingle with window history, like layout drawer. This is for the back button to be able to close them.

The good news is that v0.15 will remove this feature due to the numerous problems devs encounter. If you want some details, do a search on similar closed tickets for v0.15 where I've explained it in more details.

Bottom line, v0.15 won't have this kind of issues anymore. Releasing v0.15 this month. Thank you for your patience.

**Comment by BrianMoreno1994** [comment]
So the back button to close these components won't work? It'll just navigate back to the previous page?

**Comment by rstoenescu** [comment]
The back button feature that you got used to will still work on Cordova apps. But not on websites/PWAs.
